src/components/v5/common/ActionSidebar/partials/Motions/steps/FinalizeStep/FinalizeStep.tsx

```diff
@@ -142,9 +144,10 @@ const FinalizeStep: FC<FinalizeStepProps> = ({
           iconSize={16}
         >
           {formatText({
-            id: isMotionFailedNotFinalizable
-              ? 'motion.finalizeStep.failed.statusText'
-              : 'motion.finalizeStep.statusText',
+            id:
+              isMotionFailedNotFinalizable || (!userStake && userVote)
```

Comment:
Not sure I'm following this. Why do we want to show failed status text if user did not stake but has voted?